app/views/apparels/show.html.erb

```diff
@@ -33,7 +33,7 @@
               <% @apparel.sizes.each do |size| %>
                 <div class="size-option">  
                   <input id="size-<%= size.name %>" type="radio" name="size" value="<%= size.name %>">
-                  <label for="size-<%= size.name %>"><%= size.name %></label>
+                  <label for="size-<%= size.name %>" class="active"><%= size.name %></label>
```

Comment:
Why, are they all active?
This class is not defined in your css, you should remove it from the html